src/installer/__main__.py

```diff
@@ -0,0 +1,67 @@
+"""CLI - run as python -m installer."""
+import argparse
+
+from . import install
+from .destinations import SchemeDictionaryDestination
+from .sources import WheelFile
+
+
+def main():
+    """Entry point for CLI."""
+    ap = argparse.ArgumentParser("python -m installer")
+    ap.add_argument("wheel_file", help="Path to a .whl file to install")
```

Comment:
Does this support installing the wheel file if passed as a glob, like `dist/*.whl`?

Reply:
No, it won't be unglobbed in Python, but why would it not be unglobbed in the shell?

Reply:
Currently we invoke `python setup.py install` while in the package directory, maybe we should make the `wheel_file` argument optional and autodetect the dist folder and install from there if present and a `.whl` file is there.

Reply:
It will work if the shell expands the glob to a single wheel. At present, it won't accept multiple wheels to install - that's easy enough to do if necessary, but I wanted to start with the simplest thing.

> maybe we should make the wheel_file argument optional and autodetect...

I think the crucial 'what to install' input should be 100% explicit. That's in keeping with the general pattern of this library. If you know there's a single wheel under `dist`, and you're running it from a shell, you can pass `dist/*.whl` and let the shell expand it.

Reply:
> it won't accept multiple wheels to install

Oh, not wanting it to do that, just pick up a single wheel from dist actually.

> If you know there's a single wheel under `dist`, and you're running it from a shell, you can pass `dist/*.whl` and let the shell expand it.

Yeah, was going to do something like that otherwise, figured would be kinda nice to have similar semantics to existing install methods. Maybe have it use only the newest file in `dist/*.whl` would make sense or something if a glob is passed?